packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -134,6 +142,9 @@ export function ModelsTree(props: ModelsTreeProps) {
                   {...baseRendererProps}
                   nodeLoader={state.nodeLoader}
                   nodeRenderer={(nodeProps) => <ModelsTreeNodeRenderer {...nodeProps} density={density} />}
+                  onApplyFilterClick={() => reportUsage?.({ reportInteraction: true })}
+                  onFilterClear={() => reportUsage?.({ reportInteraction: true })}
+                  onFilterApplied={() => reportUsage?.({ featureId: "hierarchy-level-filtering", reportInteraction: true })}
```

Comment:
Other trees can also support filtering but does not report it. Maybe these three cases can be handled inside `FilterableTreeRenderer` so we just need to pass `reportUsage`?

Reply:
Moved into `FilterableTreeRenderer`.